src/libs/Notification/LocalNotification/BrowserNotifications.js

```diff
@@ -92,20 +93,35 @@ export default {
      * Create a report comment notification
      *
      * @param {Object} params
+     * @param {Object} params.report
      * @param {Object} params.reportAction
      * @param {Function} params.onClick
      * @param {Boolean} usesIcon true if notification uses right circular icon
      */
-    pushReportCommentNotification({reportAction, onClick}, usesIcon = false) {
+    pushReportCommentNotification({report, reportAction, onClick}, usesIcon = false) {
+        let title;
+        let body;
+
+        const isChatRoom = ReportUtils.isChatRoom(report);
+
         const {person, message} = reportAction;
         const plainTextPerson = _.map(person, (f) => f.text).join();
 
         // Specifically target the comment part of the message
         const plainTextMessage = (_.find(message, (f) => f.type === 'COMMENT') || {}).text;
 
+        if (isChatRoom) {
+            const roomName = _.get(report, 'displayName', '');
```

Comment:
Does this need to be `reportName`? Because I can't see the `displayName` property. I was working on this PR https://github.com/Expensify/App/pull/28200#issuecomment-1747176241 and realized that the notifications for chat rooms might be broken?

Reply:
This is a regression from https://github.com/Expensify/App/pull/21406 https://github.com/Expensify/App/pull/21406#discussion_r1284703138. This should be changed to:

```js
const roomName = ReportUtils.getReportName(report);
```

But if `reportName` prop already exists we can use it directly.

Reply:
Ok, great, thanks for the context. I will update it to use `getReportName`.